content/en/tracing/setup_overview/compatibility_requirements/dotnet-core.md

```diff
@@ -21,33 +21,57 @@ further_reading:
 
 - The .NET Tracer library for Datadog is open source. For more information, see the [.NET Tracer repository][1].
 
+## Runtime support policy for .NET Core APM
```

Comment:
As a customer, I would prefer that the first thing I see is the list of supported .NET Core runtimes and platforms/architectures. Since you link to this table from other places in the doc, can this section be placed last or closer to the bottom of the page?